rollup.config.js

```diff
@@ -33,11 +33,11 @@ module.exports = {
         clear({
             targets: ['build/'],
         }),
+        commonjs(),
```

Comment:
same here, I assume there is a deeper reason for this change that would get lost in PRs

Reply:
Yup split out into https://github.com/guardian/discussion-rendering/pull/478